tests/test_s3.py

```diff
@@ -1910,6 +1910,129 @@ def test_delete_bucket_notification(self):
         us = [t for t in topic_notifications if t.get('TopicArn') == topic_arn]
         self.assertEqual(len(us), 1)
 
+    def test_enable_bucket_encryption_kms(self):
+        session_factory = self.replay_flight_data('test_s3_enable_bucket_encryption_kms')
+        session = session_factory()
+        client = session.client('s3')
+        kms_client = session.client('kms')
+        bname = 'custodian-enable-bucket-encryption-kms'
+
+        client.create_bucket(Bucket=bname)
+        self.addCleanup(destroyBucket, client, bname)
+
+        try:
+            response = client.get_bucket_encryption(Bucket=bname)
+        except ClientError as e:
+            self.assertTrue(e)
```

Comment:
It would be more useful to assert that the call returns a not found exception instead of just asserting that there is an exception.